share/spice/piratetalk/piratetalk.js

```diff
@@ -0,0 +1,28 @@
+(function (env) {
+    "use strict";
+    env.ddg_spice_piratetalk = function(api_result) {
+
+        if (!api_result || api_result.translation==="") {
+            return Spice.failed('piratetalk');
+        }
+        
+        Spice.add({
+            id: "piratetalk",
+            name: "Answer",
+            data: {
+                title:api_result.translation,
+            },
+            meta: {
+                sourceName: "Pirate",
+                sourceUrl: 'https://pirate-speak.herokuapp.com/',
+            },
+            templates: {     
+                group: 'base',
```

Comment:
@GuiltyDolphin I basically wanted the result to look like quotes, similar to quotes spices. And the result can be a line of words sometime, didnt want to use text template.